dev/src/category.rb

```diff
@@ -95,7 +101,7 @@ def add(child)
 
     child.parent = self
     children << child
-    children.sort!
+    children.sort!.uniq!
```

Comment:
Do we have duplicates in the source data or a code path that leads to duplicates? Perhaps we should change `children`/`attributes` to be `Set`.

Reply:
I had them as set's originally, but Ruby doesn't have ordered Set's 😮‍💨

Source data _shouldn't_ have duplicates, and we do have verification of no duplicates on the other side, so this isn't strictly needed